.github/semantic.yml

```diff
@@ -0,0 +1,5 @@
+# Always validate the PR title AND all the commits
```

Comment:
where is this file used?

Reply:
This file is being used by the semantic pull requests plugin, that ensure at least one semantic commit and title for each PR: https://github.com/zeke/semantic-pull-requests#configuration